### PR TITLE
API-2215: TOTP generator in the open GitHub

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,30 @@ This is a placeholder README.md for a new repository
...
+example: `./generate-totp.sh BZDHSGCIGQXRWVQX`
+
+
+***
+
+# More info:
+* [TOTP algorithm](https://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm)
+* [TOTP algorithm specifications](https://tools.ietf.org/html/rfc6238)
+* [HOTP algorithm specifications](https://tools.ietf.org/html/rfc4226)
+* [Oauth 2.0 specifications](https://tools.ietf.org/html/rfc6749)
```
Comment (hughfdjackson) [inline README.md]: We need to give a _lot_ more context here - for instance: 

* What use case is it built for (i.e. it's for testing purposes only - not intended to be part of a prod stack)?
* How many digits does it use? (i.e. 8)
* What is the OAuth 2.0 link necessary - we haven't mentioned that it's for testing out a flow we have as part of OAuth 2.0.

File: project/HmrcBuild.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "org.pegdown" % "pegdown" % "1.5.0" % "test"
+  )
+
+  def apply() = compile ++ testDependencies
+}
+
+object Developers {
+
+  def apply() = developers := List[Developer]()
+}
```
Comment (hughfdjackson) [inline project/HmrcBuild.scala]: I believe we need license in all code files.

File: README.md [hunk 3]
```diff
@@ -14,12 +14,16 @@ This code is open source software licensed under the [Apache 2.0 License]("http:
 
 # Instructions
 
-Tool that generates a TOTP (timed-based-one-time password) from a secret.
-It uses the HMAC-SHA512 encryption
+This tool generates a 8-digits TOTP code (Timed Based One-time Password) from a shared secret key.
+Thd implementation uses a HMAC (Hash-based Message Authentication Code) SHA-512 encryption algorithm.
```
Comment (RawToast) [inline README.md]: Typo "The implementation"

File: project/HmrcBuild.scala (hunk 2 above)
Comment (gokyo) [inline project/HmrcBuild.scala]: Done

File: README.md (hunk 3 above)
Comment (gokyo) [inline README.md]: done

**Comment by gokyo** [comment]
@hughfdjackson 
I updated the README

File: README.md (hunk 1 above)
Comment (gokyo) [inline README.md]: The readme has been updated now

File: project/HmrcBuild.scala (hunk 2 above)
Comment (edgarjimenez) [inline project/HmrcBuild.scala]: Can we please reduce the number of commits? Other than that, it looks fine.

**Pull request merged**